pkg/deploy/elbv2/listener_manager.go

```diff
@@ -156,6 +156,13 @@ func (m *defaultListenerManager) updateSDKListenerWithSettings(ctx context.Conte
 // currentExtraCertificates is the current extra certificates, if it's nil, the current extra certificates will be fetched from AWS.
 func (m *defaultListenerManager) updateSDKListenerWithExtraCertificates(ctx context.Context, resLS *elbv2model.Listener,
 	sdkLS ListenerWithTags, isNewSDKListener bool) error {
+	// if TLS is not supported, we shouldn't update
+	if sdkLS.Listener.SslPolicy == nil {
+		m.logger.Info("SDK Listner doesn't have SSL Policy set, we skip updating extra certs.")
```

Comment:
consider modifying the log message 
"skip updating extra certs for non-TLS listener"
also increase verbosity - m.logger.V(1).Info